crawler/attribution/entrypoints/blob-url.js

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Not that I really care, that's the justification for the { in these files?

Reply:
This only happens with var right?

Reply:
In this case top-level `const` variables do collide (i'm getting an exception). I think these scripts share the same scope or something like that...